src/App.js

```diff
@@ -13,7 +13,7 @@ function App() {
       <Splash />
       <About />
       <Events />
-      <img src='/divider.jpg' alt='code on a screen. mostly css' />
+      <img id="divider" src="/divider.jpg" alt="code on a screen. mostly css"/>
```

Comment:
We should follow the css standards of other `img` tags in our source using the `className` attribute instead of `id`. [example](https://github.com/FCCColumbus/cbus-web/blob/dfdbbe7925ec08812d37fc6a58a5699fa03a2ffc/src/components/Members.jsx#L38)